index.html

```diff
@@ -231,5 +231,97 @@ <h3 class="proj__title">Skills</h3>
         </article>
       </div>
     </main>
+
+    <footer class="contact__form">
+      <div class="contact__title">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+      <form
+        class="form__collab"
+        action="https://formspree.io/f/xknepwnz"
+        method="POST"
+        id="form__all"
+      >
+        <ul class="form__list">
+          <li class="form__input">
+            <label>
+              <input
+                placeholder="Full Name"
+                type="text"
+                id="name"
+                name="name"
+                minlength="3"
+                maxlength="30"
+                required
+                pattern="([A-ZΆ-ΫÀ-ÖØ-Þ][A-ZΆ-ΫÀ-ÖØ-Þa-zά-ώß-öø-ÿ]{1,19} ?){1,10}"
+                onchange="storeForm()"
+              />
+            </label>
+          </li>
+
+          <li class="form__input">
+            <label>
+              <input
+                placeholder="Email address"
+                class="email_bg"
+                id="mail"
+                type="email"
+                placeholder="email@domain.com"
+                name="email"
+                required
+              />
+            </label>
+          </li>
+          <ul class="txt_area">
+            <textarea
+              name="Message input"
+              id="msg"
+              placeholder="Enter your message here..."
+              maxlength="500"
+              required
+            ></textarea>
+            <li><p class="success_message" id="error_message"></p></li>
+            <li class="align_center">
+              <button class="collab__btn" id="fot" type="submit">
+                Get in touch
+              </button>
```

Comment:
I think your button is looking different from the given design🤗, kindly fix it as it is a requirement for this milestone that you stick to the design🤗

your application
<img width="269" alt="your_get_______Touch" src="https://user-images.githubusercontent.com/64643995/213020100-e9d7afa3-1578-41f0-a598-c009296495e3.png">

the design
<img width="188" alt="get______________touch" src="https://user-images.githubusercontent.com/64643995/213020174-345cf8b7-aff0-4f05-9c8f-4acd30beccb6.png">